app/views/grammars/_form.html.erb

```diff
@@ -13,7 +13,7 @@
 
   <div class="field">
     <%= form.label :dialect_id %>
-    <%= form.text_field :dialect_id %>
+    <%= form.select :dialect_id, Dialect.order(:name_en).pluck(:name_en, :id) %>
```

Comment:
One more thing. Can you add `include_blank: true` to the end of the `select`. 

Should look like this
`form.select :dialect_id, Dialect.order(:name_en).pluck(:name_en, :id), include_blank: true`

Reply:
Done.